src/test/java/org/opentripplanner/apis/transmodel/mapping/RequestModesMapperTest.java

```diff
@@ -0,0 +1,64 @@
+package org.opentripplanner.apis.transmodel.mapping;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+import java.util.HashMap;
+import java.util.Map;
+import org.junit.jupiter.api.Test;
+import org.opentripplanner.routing.api.request.RequestModes;
+import org.opentripplanner.routing.api.request.StreetMode;
+
+class RequestModesMapperTest {
+
+  @Test
+  void testMapRequestModesEmptyMapReturnsDefaults() {
+    Map<String, StreetMode> inputModes = new HashMap<>();
+
+    RequestModes mappedModes = RequestModesMapper.mapRequestModes(inputModes);
+
+    assertEquals(RequestModes.of().build(), mappedModes);
+  }
+
+  @Test
+  void testMapRequestModesAccessSetReturnsDefaultsForOthers() {
+    Map<String, StreetMode> inputModes = new HashMap<>();
+
+    inputModes.put("accessMode", StreetMode.BIKE);
```

Comment:
```suggestion
    Map<String, StreetMode> inputModes = Map.of("accessMode", StreetMode.BIKE);
```

Not that it matters much, but we tend to write it this way. Totally up to you.

Reply:
Done.